### DISCUSSION TITLE
How to show/hide menuBar dynamically

### DISCUSSION THREAD

**Discussion by sh977218** [primefaces/community#1082] [PrimeNG]
I see there is a property `visible` in the menuBar's config. But this is not two way data binding. i.e. if I set this variable in OnInit, and this variable changes later, the change is not reflected.
Should we pass any function instead of boolean for this property?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can clearly identify the issue?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity
